ios/Social Distance Alarm/SocialDistanceSDK/SocialDistanceSDK.xcodeproj/project.pbxproj

```diff
@@ -392,7 +392,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = SocialDistanceSDK/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
@pzmudzinski The restriction to iOS 13 was intentional.  The handsets that don't support iOS 13 also don't have hardware that supports BLE.